### DISCUSSION TITLE
How to install mods

### DISCUSSION THREAD

**Discussion by Dal1ster** [KadeArchive/Kade-Engine#407] [Q&A] [answered]
I am trying to install this mod, but when I start up the kade engine, it doesn't appear in the week selection screen nor the freeplay screen. What do i do? [https://cdn.discordapp.com/attachments/637714058421993493/840392002529001492/Ugh.rar](url)
Source from silvagunner video: [https://www.youtube.com/watch?v=c_6rt9Tj5aI&t=1s](url)

**Comment by puyoxyz** [comment, marked as answer]
you can't install a week 7 mod because week 7 doesn't exist in kade engine yet

**Comment by dacixn** [comment]
standalone friday night funkin' and kade engine have different file systems i'm pretty sure?

**Reply by puyoxyz** [reply]
No, what you know as the Kade Engine file structure is the week 7 file structure. So, starting with 0.2.8 (week 7) of Friday Night Funkin, both will have the same file structure.

**Comment by KlavierGayming** [comment]
Normal Kade Engine doesn't have week 7. You need to use the deleted week 7 pre-release from kade.
